python/ray/train/v2/api/config.py

```diff
@@ -112,9 +112,13 @@ class RunConfig(RunConfigV1):
         checkpoint_config: Checkpointing configuration.
         callbacks: [DeveloperAPI] A list of callbacks that the Ray Train controller
             will invoke during training.
+        worker_runtime_env: [DeveloperAPI] Runtime environment configuration
+            for each Ray Train worker actor.
     """
 
     callbacks: Optional[List["UserCallback"]] = None
+    worker_runtime_env: Optional[dict] = None
```

Comment:
Should also take/support [ray.runtime_env.RuntimeEnv](https://docs.ray.io/en/latest/ray-core/api/doc/ray.runtime_env.RuntimeEnv.html)